docs/source/build.md

```diff
@@ -57,12 +52,6 @@ mamba install -c rapidsai -c conda-forge -c nvidia libraft-headers cuda-version=
 
 `pylibraft` and `raft-dask` both have packages that can be [installed through pip](https://rapids.ai/pip.html#install).
 
-For CUDA 11 packages:
-```bash
-pip install pylibraft-cu11 --extra-index-url=https://pypi.nvidia.com
-pip install raft-dask-cu11 --extra-index-url=https://pypi.nvidia.com
-```
-
 And CUDA 12 packages:
```

Comment:
nit: the 'And' in this line is dangling - maybe we should just delete this line

Reply:
Let me fix that in 25.10.